api/v1beta1/disruption_webhook.go

```diff
@@ -62,6 +62,7 @@ func (r *Disruption) SetupWebhookWithManager(setupWebhookConfig utils.SetupWebho
 	handlerEnabled = setupWebhookConfig.HandlerEnabledFlag
 	defaultDuration = setupWebhookConfig.DefaultDurationFlag
 	cloudServicesProvidersManager = setupWebhookConfig.CloudServicesProvidersManager
+	cloudServicesProvidersManager = setupWebhookConfig.CloudServicesProvidersManager
```

Comment:
Not sure it's needed to have this assignation twice, the line above already does the same thing?

Reply:
sorry, this is a merging mistake 😓